### ISSUE TITLE
transparent/translucent versions

### ISSUE THREAD

**Issue by VoidNoire** [KaizIqbal/Bibata_Cursor#27]
**Is your feature request related to a problem? Please describe.**
i don't want the cursor to (completely) cover whatever is behind it.

**Describe the solution you'd like**
i'd like a version of it that only has the white outlines or a version where the white outlines are opaque but the other colours are translucent (maybe have a couple versions that each have different degrees of transparency?).

**Describe alternatives you've considered**
i've tried using [phoeton cursor](https://www.gnome-look.org/p/999964/) which looks nice but is incomplete.

**Comment by Silicasandwhich** [comment]
Exactly how translucent do you want them to be? I'm currently making a translucent cursor flavor and it would be really annoying to redo it all.

**Comment by VoidNoire** [comment]
> Exactly how translucent do you want them to be? I'm currently making a translucent cursor flavor and it would be really annoying to redo it all.

personally, i think around 70% opacity would be nice but i don't really have a strong preference. i'm just glad someone is working on a translucent version already. i'd say it's probably best to finish the one you're already working on since it sounds like you've already made a lot of progress on it anyways.

just fwiw, if you plan to make other versions with different degrees of opacity, it might be a good idea to write a script that takes opacity as input and automatically generates the images with different opacities depending on the input.

**Comment by Silicasandwhich** [comment]
It may be a little hard to script something like that. The way the original cursors were designed requires me to delete some parts and make new ones, cut out major sections of parts of the cursors, and sometimes even remake the shadows.

My current opacity level is around 60% which I've found looks pretty snazzy. I probably won't be making different versions with different degrees of opacity since I'm making three sets of cursors for just one variation, but it might be one of my projects after I've finished with the first set.

Oh, and if anybody knows what font @KaizIqbal used in the gallery images, that would be a huge help.
